content/docs/studio/selfhosted/installation/helm.md

```diff
@@ -0,0 +1,109 @@
+# Installing Studio Self-hosted via Helm
+
+## Prerequisites
+
+**Software**
+
+You can deploy Studio Self-hosted on any Kubernetes cluster (version => 1.20)
+that has an ingress controller configured and installed.
+
+The choice of ingress controller is up to you. We have good experiences with
+[ingress-nginx](https://kubernetes.github.io/ingress-nginx/) but Studio will
+work on other ingress controllers as well.
+
+You'll also need to have the following CLI tools configured and installed:
+
+- Helm
+- Kubectl
+
+**DNS**
+
+Create the following two DNS records pointing to the IP address of your load
+balancer:
```

Comment:
That's confusing, we're not writing explicitly we require a load balancer of any kind